cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.cpp

```diff
@@ -146,9 +150,9 @@ auto getUniformsFromViewport(const std::shared_ptr<Viewport>& viewport, double d
   // Memoized in the JS
   auto uniforms = calculateViewportUniforms(viewport, devicePixelRatio, coordinateSystem, coordinateOrigin);
 
-  uniforms.project_uWrapLongitude = wrapLongitude;
+  uniforms.uWrapLongitude = wrapLongitude;
   // elided IDENTITY_MATRIX default
-  uniforms.project_uModelMatrix = viewport->modelMatrix;
+  uniforms.uModelMatrix = viewport->modelMatrix;
```

Comment:
+1 I think we can also drop the `u` Prefix now. It was used to signify this is a uniform, but again it becomes unnecessary since this is now a field in a struct.